synthesizer/src/vm/finalize.rs

```diff
@@ -1048,7 +1050,7 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                     //     }
                     // }
 
-                    // Calculate the stake per validator using `bonded_balances`.
+                    // Calculate the stake per validator using `bonded_balances`. No need to check delegated mapping as this is the genesis block.
```

Comment:
For safety, we could assert that the total bonded balance matches the total delegated balance, but as this is the genesis block, we shouldn't have arbitrary bonded balances to addresses not in the committee